airflow/jobs/scheduler_job.py

```diff
@@ -1486,13 +1486,16 @@ def _process_executor_events(self, simple_dag_bag, session=None):
         """
         Respond to executor events.
         """
+        ti_key_to_try_number_map = {}
```

Comment:
It's not "reall" TI key.

Reply:
I know but I wanted to avoid `ti_key_without_try_number_to_try_number_map` 😄 What about `short_ti_key`? I think we can add this type to task_instance as I think we are using it in few places

Reply:
What do you think about ti_primary_key?

Reply:
Nice one 👌